testsuite/src/main/resources/jade/views/includes/footer.jade

```diff
@@ -1,2 +1,2 @@
 #footer
-  p.footer Copyright (c) Ivar Grimstad
\ No newline at end of file
+  p.footer Copyright (c) Eclipse Krazo committers and contributors
```

Comment:
The integration tests checks a copyright header? That's - creative... 😄

But it reminds me that most of the templates from the test suite and the the examples don't have a copyright header at all - is that a problem?

Reply:
No, I think that's not a problem at all. I guess adding copyright headers source files is required, but for most other files I don't think that this is really necessary.

Reply:
The jade template is just a weird exception to that. And the fact that the test checks the content of the footer actually caused some "surprising results" when I changed the headers the first time. :laughing:

Reply:
It doesn't check the copyright. It checks if a jade`include` is working.